Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:4.0
+// swift-tools-version:5.0
```

Comment:
Let's keep this 4.0 so that earlier versions of Swift can use the package.
```suggestion
// swift-tools-version:4.0
```